modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -10,14 +10,19 @@ Review the following limitations on support for local registries:
 [IMPORTANT]
 You cannot use local registries during the installation of OpenShift operator. However, you can use local registries with OpenShift RTF to store Mule Docker images and install and manage Runtime Fabric for running Mule apps. 
 
-* You must use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or later. Note that `rtfctl` installs Runtime Fabric agent to the latest version by default.
+* You can use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or later. Note that `rtfctl` installs Runtime Fabric agent to the latest version by default.
+* You can use Helm to install Rutime Fabric and configure your local registry.
 * You can use a local registry only if you create a new Runtime Fabric instance of version 1.12.0 or later.
 +
 If you set up a local registry, you can upgrade to a version later than 1.12.0 without reinstalling Runtime Fabric.
 +
 You cannot add a local registry to any version of an existing Runtime Fabric instance, even if you upgrade that installation to 1.12.0 or later.
 * If you create a Runtime Fabric instance to use with a local registry, you cannot later reconfigure it to use the `rtf-runtime-registry` endpoint. To use the `rtf-runtime-registry` endpoint, you must install a new Runtime Fabric instance using the standard installation procedure.
-* You're responsible for synchronization between your registry and the Runtime Fabric registry. MuleSoft doesn't provide additional software to synchronize images between the registries. Push all the dependency images for your version to your local registry before you start the installation. See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes-2.x.x.adoc[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version.  
+* You're responsible for synchronization between your registry and the Runtime Fabric registry. 
```

Comment:
```suggestion
* You're responsible for the synchronization between your registry and the Runtime Fabric registry. 
```

Reply:
More active wording: "You're responsible for synchronizing your registry with the Runtime Fabric registry."